dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -5557,6 +5564,16 @@ public boolean isInodeIndexedArchived(String inode){
         return isInodeIndexedWithQuery("+inode:" + inode + String.format(" +deleted:%s",true));
```

Comment:
I think the existing `isInodeIndexedWithQuery` could call the new `isInodeIndexedArchived` you made.